### ISSUE TITLE
`validator info` returns incorrect values for `liquidContract` , `liquidSupply` and `totalSlashed`. `selfUnbondingStakeLocked` missing

### ISSUE THREAD

**Issue by cmjc** [autonity/aut#125]
_See also following comment where a second bug was found after bonding self-bonded and delegated stake_

**Bug**
The command `validator info` is returning the zero address in the `liquidContract` field instead of the actual validator liquid contract address.

**Expected behaviour**
The correct address is returned.

**aut version**

```console
$ aut --version
aut, version 0.3.0.dev1
```

**Example**

```json
aut validator info --validator 0x21bb01Ae8EB831fFf68EbE1D87B11c85a766C94C
{
  "treasury": "0x61EE7d3244642E5f6D654416a098DEabFBF5306e",
  "node_address": "0x21bb01Ae8EB831fFf68EbE1D87B11c85a766C94C",
  "oracle_address": "0x9b844631B7279576330B9B822bE79266696fF8C2",
  "enode": "enode://b2748268c31ebab8603058335bb4bed062e05b9ceaa3562f69868a01d1038a84136fc587fb913e1cb8ce821f1eb0bf9879e3249f18adcd39f1211a104ceb57a9@35.197.223.249:30303",
  "commission_rate": 1000,
  "bonded_stake": 10000000000000000000000,
  "unbonding_stake": 0,
  "unbonding_shares": 0,
  "self_bonded_stake": 10000000000000000000000,
  "self_unbonding_stake": 0,
  "self_unbonding_shares": 0,
  "liquid_contract": "0x0000000000000000000000000000000000000000",
  "liquid_supply": 1397840815523076466699159265359708166239426845751,
  "registration_block": 0,
  "total_slashed": 0,
  "jail_release_block": 0,
  "provable_fault_count": 0,
  "state": 0
}
```

Using an RPC curl the actual value is correctly returned:

```rpc

curl --silent --location --request GET 'http://127.0.0.1:8545' --header 'Content-Type: application/json' --data-raw '{
                "jsonrpc":"2.0",
                "method":"aut_getValidator", "params":["0x21bb01Ae8EB831fFf68EbE1D87B11c85a766C94C"],
                "id":1}' | jq
{
  "jsonrpc": "2.0",
  "id": 1,
  "result": {
    "treasury": "0x61ee7d3244642e5f6d654416a098deabfbf5306e",
    "nodeAddress": "0x21bb01ae8eb831fff68ebe1d87b11c85a766c94c",
    "oracleAddress": "0x9b844631b7279576330b9b822be79266696ff8c2",
    "enode": "enode://b2748268c31ebab8603058335bb4bed062e05b9ceaa3562f69868a01d1038a84136fc587fb913e1cb8ce821f1eb0bf9879e3249f18adcd39f1211a104ceb57a9@35.197.223.249:30303",
    "commissionRate": 1000,
    "bondedStake": 1e+22,
    "unbondingStake": 0,
    "unbondingShares": 0,
    "selfBondedStake": 1e+22,
    "selfUnbondingStake": 0,
    "selfUnbondingShares": 0,
    "selfUnbondingStakeLocked": 0,
    "liquidContract": "0xf4d9599afd90b5038b18e3b551bc21a97ed21c37",
    "liquidSupply": 0,
    "registrationBlock": 0,
    "totalSlashed": 0,
    "jailReleaseBlock": 0,
    "provableFaultCount": 0,
    "state": 0
  }
```
**To replicate**

1. call `aut validator list` to return the list of validators.
2. call `aut validator info` on one of the validators
3. returned `liquidContract` value is the zero address
4. repeat using a `curl` - actual `liquidContract` address is returned

**Comment by cmjc** [comment]
Having registered a validator, I now find it is also returning an incorrect `liquidSupply` value and `totalSlashed` value.

At this point the validator is registered, but has no bonded stake:

```
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut validator info --validator 0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B
{
  "treasury": "0xF47FDD88C8f6F80239E177386cC5AE3d6BCdEeEa",
  "node_address": "0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B",
  "oracle_address": "0xFd91928d58Af4AFbD78C96821D3147ef1f517072",
  "enode": "enode://a3b821f89d8ea172421dedacdb00e76d8d6a929e4c5ff3c2b30ec84144a7405698ce30ba2ed482770ad2df94d050311b350c036d7f2cef3c9ef32be3f635d62e@51.89.151.55:30303",
  "commission_rate": 1000,
  "bonded_stake": 0,
  "unbonding_stake": 0,
  "unbonding_shares": 0,
  "self_bonded_stake": 0,
  "self_unbonding_stake": 0,
  "self_unbonding_shares": 0,
  "liquid_contract": "0x0000000000000000000000000000000000000000",
  "liquid_supply": 245822994132721560041932967934037774519317912847,
  "registration_block": 0,
  "total_slashed": 573883,
  "jail_release_block": 0,
  "provable_fault_count": 0,
  "state": 0
}
```

Now I bond 1 NTN from Alice (validator operator, self-bonded stake) and 0.5 NTN from Bob (delegator, delegated stake):

```
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut validator bond --validator 0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B 1 | aut tx sign - | aut tx send -
(consider using 'KEYFILEPWD' env var).
Enter passphrase (or CTRL-d to exit): 
0x2f80ce1539d8eb688b5ed953b20d1f72a0967abfcee3934f87c7b1d4b88a6b38
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ nano .autrc
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut validator bond --validator 0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B 0.5 | aut tx sign - | aut tx send -                                                                                                                    
(consider using 'KEYFILEPWD' env var).
Enter passphrase (or CTRL-d to exit): 
0x59d05fd010ba67ac9fdd3c02874bc99fa0f8315a4d8db66f81de0bc6282600bb
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut validator info --validator 0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B                      
{
  "treasury": "0xF47FDD88C8f6F80239E177386cC5AE3d6BCdEeEa",
  "node_address": "0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B",
  "oracle_address": "0xFd91928d58Af4AFbD78C96821D3147ef1f517072",
  "enode": "enode://a3b821f89d8ea172421dedacdb00e76d8d6a929e4c5ff3c2b30ec84144a7405698ce30ba2ed482770ad2df94d050311b350c036d7f2cef3c9ef32be3f635d62e@51.89.151.55:30303",
  "commission_rate": 1000,
  "bonded_stake": 0,
  "unbonding_stake": 0,
  "unbonding_shares": 0,
  "self_bonded_stake": 0,
  "self_unbonding_stake": 0,
  "self_unbonding_shares": 0,
  "liquid_contract": "0x0000000000000000000000000000000000000000",
  "liquid_supply": 245822994132721560041932967934037774519317912847,
  "registration_block": 0,
  "total_slashed": 573883,
  "jail_release_block": 0,
  "provable_fault_count": 0,
  "state": 0
}
```

After the staking transition has been applied:

```
ubuntu@vps-c7c3e8c7:~/TEST/autcli$ aut validator info --validator 0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B
{
  "treasury": "0xF47FDD88C8f6F80239E177386cC5AE3d6BCdEeEa",
  "node_address": "0xa418A886fbE5E08016d59AD90FC84fEe912D7A5B",
  "oracle_address": "0xFd91928d58Af4AFbD78C96821D3147ef1f517072",
  "enode": "enode://a3b821f89d8ea172421dedacdb00e76d8d6a929e4c5ff3c2b30ec84144a7405698ce30ba2ed482770ad2df94d050311b350c036d7f2cef3c9ef32be3f635d62e@51.89.151.55:30303",
  "commission_rate": 1000,
  "bonded_stake": 1500000000000000000,
  "unbonding_stake": 0,
  "unbonding_shares": 0,
  "self_bonded_stake": 1000000000000000000,
  "self_unbonding_stake": 0,
  "self_unbonding_shares": 0,
  "liquid_contract": "0x0000000000000000000000000000000000000000",
  "liquid_supply": 245822994132721560041932967934037774519317912847,
  "registration_block": 500000000000000000,
  "total_slashed": 573883,
  "jail_release_block": 0,
  "provable_fault_count": 0,
  "state": 0
}
```

**Comment by aiman** [comment]
Thanks, fixed in v0.3.0.dev2
